docs/Operators.md

```diff
@@ -6329,12 +6329,14 @@ expect(node, inputs=[x], outputs=[y],
         const int max_trip_count = 10; // Analogous to input M
         int user_defined_vals[]; // Imagine this is resizable
         /* End implicitly-defined code */
+        int b_out = b;
         for (int i=0; i < max_trip_count && keepgoing; ++i) {
           /* User-defined code (loop body) */
           int my_local = a + b; // Reading values in the enclosing scope is fine
-          b = a - b; // writes fine if we specify b as a loop-carried dependency
-          keepgoing = my_local > b; // keepgoing is a loop-carried dependency
+          b_out = a - b; // writes fine if we specify b as a loop-carried dependency
+          keepgoing = my_local > b_out; // keepgoing is a loop-carried dependency
           user_defined_vals[i] = b + b;
+          b = b_out;
```

Comment:
With `b_out`, `b` inside the loop body can probably be removed. I personally like changing `b_out` below to `b` or adding `b_out=b` right after this loop.

Reply:
I think we can use "b" uniformly inside the loop, and add b_out=b after the loop. (See suggested change to the ONNX example discussed above.)

Reply:
Look like we need to add `keepgoing_out = keepgoing` right after the loop too.